### PR TITLE
DO NOT MERGE: Iiif-v3plus5

### PR DESCRIPTION
**Pull request by ShanaLMoore**
Changes were made for [iiif_print's repo](https://github.com/scientist-softserv/iiif_print/pull/46).

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -94,7 +94,6 @@ jobs:
       - name: Start containers
         run: >-
           cd ${{ inputs.subdir }};
-          chmod 777 db/schema.rb;
```
Comment (orangewolf) [inline .github/workflows/test.yaml:97]: use chmod -f which will skip if the file is missing

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -70,8 +70,6 @@ jobs:
           image_name: ${{ inputs.image_name }}
       - name: Checkout code
         uses: actions/checkout@v3
-        with:
```
Comment (orangewolf) [inline .github/workflows/build.yaml:73]: this should be fine in the v3plus5

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -109,11 +108,6 @@ jobs:
           else
             echo "No solr directory found, skipping..."
           fi
-      - name: Setup db
```
Comment (orangewolf) [inline .github/workflows/test.yaml:112]: maybe an if statement for this one?

Comment (ShanaLMoore) [reply .github/workflows/test.yaml:112]: I'd need to check for the existence of a database

**Pull request closed without merging**